dockerfiles/taraxa_builder.dockerfile

```diff
@@ -0,0 +1,49 @@
+# Taraxa remote docker environment - it is intended to be used for taraxa-node building
+#
+# Build and run:
+#
+#   DOCKER_BUILDKIT=1 docker build --progress=plain -t taraxa/taraxa_builder:1.0 -f taraxa_builder.dockerfile .
+#   docker run -ti --name taraxa_builder taraxa/taraxa_builder:1.0 /bin/bash
+#
+# stop:
+#   docker stop taraxa_builder
+#
+# users:
+#   root, passwd: root
+
+FROM ubuntu:20.04
+
+# Set password for root -> root
+RUN echo "root:root" | chpasswd
+
+# Install standard packages
+RUN apt-get update
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends tzdata
+RUN apt-get install -y \
+      nano \
+      tar \
+      git \
+      wget \
+      cmake ccache gcc g++ clang clang-format clang-tidy gdb \
+      libboost-program-options-dev libboost-system-dev libboost-filesystem-dev libboost-thread-dev libboost-log-dev \
+      libtool \
+      autoconf \
+      libssl-dev \
+      libjsoncpp-dev libjsonrpccpp-dev \
+      libscrypt-dev \
+      librocksdb-dev \
+      libmpfr-dev \
+      libgmp3-dev
+RUN apt-get clean
```

Comment:
Unify the `RUN` commands like this:
```
RUN apt-get update && apt-get install -y \
.......
  && rm -rf /var/lib/apt/lists/*
```
source: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run